src/nvme/mi.c

```diff
@@ -21,8 +21,7 @@
 #include "mi.h"
 #include "private.h"
 
-static const int default_timeout = 1000; /* milliseconds; endpoints may
-					    override */
+static const int default_timeout = 1000; /* milliseconds; endpoints may override */
```

Comment:
Please do not change the formatting if there is no good reason for it. It perfectly fine to use to 80 chars limits. There is no fix rule we nee do to use 100 chars.